settings.yaml.template

```diff
@@ -22,5 +22,5 @@ PROVIDERS:
   EC2:
     USERNAME:
     PASSWORD:
-    REGION:
+    REGION: []
```

Comment:
I suggest to keep the same convention for all cloud providers. Currently, the `REGION` is just a string. Now, it will be a string for Azure and GCP and an array of string for AWS.

What do you think about using `REGION` for a single string and `REGIONS` for an array? `REGIONS` could throw an error something like 'Not yet supported for Azure/GCP' for cloud providers other than AWS. In the future, others can add such a support for Azure and GCP.

Reply:
Since deleting resources from multiple regions is the objective I think an array for `REGIONS` will be a good idea.
We can also add the same support for AZURE/GCP and handle the errors accordingly in the future.
OR 
I can also create a new module which can have the Regions data and settings.yaml would then contain `REGIONS` as Strings itself.
So users would also have to update that new module file with the regions they want the resources to be deleted which will be a new task in addition to updating settings.yaml
what do you think? @apodhrad , any views on this @jyejare